### PR TITLE
delegation history

### PR DESCRIPTION
**Pull request by nicarq**
Screen Shot 2021-01-23 at 12 12 37 AM

### CODE REVIEW

**Comment by nicarq** [comment]
@SebastienGllmt @mmahut plz accept or review

**Review by mmahut — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -189,6 +189,38 @@ We recommend querying using payment key hashes (`addr_vkh`) when possible (other
...
   }
+ ```
+</details>
+<details>
+  <summary>api/pool/delegationHistory</summary>
+  Input
+
+  ```js
+  {
+    poolRanges: Dictionary<string, {fromEpoch: number, toEpoch?: number}> // operator key (pool id)
```
Comment (SebastienGllmt) [inline README.md:200]: Should probably mention whether or not these are inclusive

Comment (nicarq) [reply README.md:200]: yeah good suggestion (for the record both of them are)

**Comment by nicarq** [comment]
@mmahut merged :) could u deploy whenever u can

**Comment by mmahut** [comment]
> @mmahut merged :) could u deploy whenever u can

It is on testnet since yesterday (with new db-sync as well), will deploy to mainnet later today.

**Comment by nicarq** [comment]
@mmahut thank you!

**Pull request merged**